### DISCUSSION TITLE
405 build error for adapter-static

### DISCUSSION THREAD

**Discussion by pierreyoda** [sveltejs/kit#7171] [Q&A] [answered]
Hi there :wave:

I have a seemingly unsolvable error at build with SvelteKit and `adapter-static` (latest versions).

```
Error: 405 /layout
    at file:///Users/pyd/Projects/praca/praca-website/node_modules/@sveltejs/kit/src/core/prerender/prerender.js:50:11
    at save (file:///Users/pyd/Projects/praca/praca-website/node_modules/@sveltejs/kit/src/core/prerender/prerender.js:371:4)
    at visit (file:///Users/pyd/Projects/praca/praca-website/node_modules/@sveltejs/kit/src/core/prerender/prerender.js:235:3)

Node.js v18.9.1
[vite-plugin-svelte-kit] Prerendering failed with code 1
error during build:
Error: Prerendering failed with code 1
    at ChildProcess.<anonymous> (file:///Users/pyd/Projects/praca/praca-website/node_modules/@sveltejs/kit/src/exports/vite/index.js:438:15)
    at ChildProcess.emit (node:events:513:28)
    at ChildProcess._handle.onexit (node:internal/child_process:291:12)
```

The repository is at [praca-website](https://github.com/pierreyoda/praca-website/tree/fix-static-build), on the `fix-static-build` branch.

Reproducing the error can be done by simply doing `yarn build`.

I've looked at the SvelteKit source code and cannot manage to actually find the source of the error. Trying many combinations of `+server.ts` and/or `+layout.server.ts` did not solve the issue.

I really hope someone can help me since I really enjoy using Svelte(Kit) and cannot publish my personal website anymore (I use Vercel).

Thanks by advance!

**Comment by teemingc** [comment, marked as answer]
The error is coming from [`src/routes/layout/+server.ts`](https://github.com/pierreyoda/praca-website/blob/fix-static-build/src/routes/layout/%2Bserver.ts).

What is the purpose of this file? It isn't exporting any endpoints, so I think it should be deleted.

**Comment by pierreyoda** [comment]
Thank you so much @s3812497! It works perfectly now.

Have a great day.
